optuna/cli.py

```diff
@@ -126,6 +126,17 @@ def get_parser(self, prog_name):
         parser.add_argument('--out', '-o',
                             help='Output HTML file path. If it is not given, a HTTP server starts '
                                  'and the dashboard is served.')
+        parser.add_argument('--allow-websocket-origin',
+                            dest='bokeh_allow_websocket_origins',
+                            action='append',
+                            default=[],
+                            help='Allow websocket access from the specified host(s).'
+                                 'Internally, it is used as the value of bokeh\'s '
+                                 '--allow-websocket-origin option. Please refer to '
+                                 'https://bokeh.pydata.org/en/latest/docs/'
+                                 'reference/command/subcommands/serve.html '
+                                 'for more details.'
+                                 'access')
```

Comment:
Unintentional line?

Reply:
Sorry, this is unintentional line.
I remove it.